src/containers/Editor/components.tsx

```diff
@@ -230,10 +236,9 @@ const EditorContainer: React.FC<EditorContainerProps> = ({
   }
 
   const isReadmeEditor = active.type === 4;
-  const isCodeEditor = !isReadmeEditor;
 
   return (
-    <MainRoot>
+    /*    <MainRoot>
```

Comment:
Should remove this commented code?

Reply:
Absolutely! Pushed the commit with those changes just now.